dao/pom.xml

```diff
@@ -97,5 +92,14 @@
             <artifactId>mockito-junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
```

Comment:
No need to add this dependency. This dependency was included in testcontainers:
![image](https://user-images.githubusercontent.com/84317209/174675086-23250167-ffba-49b5-8873-89d28c3f3479.png)